Configuration/AlCa/python/autoCond.py

```diff
@@ -31,16 +31,20 @@
     'run2_data_relval'             : '122X_dataRun2_relval_v1',
     # GlobalTag for Run2 HI data
     'run2_data_promptlike_hi'      : '122X_dataRun2_PromptLike_HI_v1',
-    # GlobalTag for Run3 HLT: it points to the online GT
-    'run3_hlt'                     : '122X_dataRun3_HLT_v4',
     # GlobalTag with fixed snapshot time for Run2 HLT RelVals: customizations to run with fixed L1 Menu
     'run2_hlt_relval'              : '122X_dataRun2_HLT_relval_v1',
+    # GlobalTag for Run3 HLT: it points to the online GT
+    'run3_hlt'                     : '122X_dataRun3_HLT_v4',
+    # GlobalTag with fixed snapshot time for Run3 HLT RelVals: customizations to run with fixed L1 Menu
+    'run3_hlt_relval'              : '122X_dataRun3_HLT_relval_v1',
     # GlobalTag for Run3 data relvals (express GT)
     'run3_data_express'            : '122X_dataRun3_Express_v3',
     # GlobalTag for Run3 data relvals
     'run3_data_prompt'             : '122X_dataRun3_Prompt_v3',
     # GlobalTag for Run3 offline data reprocessing
     'run3_data'                    : '122X_dataRun3_v3',
+    # GlobalTag for Run3 data relvals: allows customization to run with fixed L1 menu
+    'run3_data_relval'             : '122X_dataRun3_relval_v1',
```

Comment:
so the snapshot of this GT is  2022-04-25 16:17:55 (UTC), see browser

Reply:
after more careful thinking, probably not, none of the PCL tags should be included in the "offline" GT, and the O2Os were not moved yet.

Reply:
Changed in 5306e7f to use fixed snapshot at 12_2_X last run, see (updated) PR description.